### ISSUE TITLE
MobileNets: Efficient Convolutional Neural Networks for Mobile Vision Applications

### ISSUE THREAD

**Issue by guanfuchen** [guanfuchen/DeepNetModel#6]
related paper

|摘要|
|---|
|We present a class of efficient models called MobileNets for mobile and embedded vision applications. MobileNets are based on a streamlined architecture that uses depthwise separable convolutions to build light weight deep neural networks. We introduce two simple global hyperparameters that efficiently trade off between latency and accuracy. These hyper-parameters allow the model builder to choose the right sized model for their application based on the constraints of the problem. We present extensive experiments on resource and accuracy tradeoffs and show strong performance compared to other popular models on ImageNet classification. We then demonstrate the effectiveness of MobileNets across a wide range of applications and use cases including object detection, finegrain classification, face attributes and large scale geo-localization.|

**Comment by guanfuchen** [comment]
applications

![image](https://user-images.githubusercontent.com/22321977/48877315-b57bb800-ee3d-11e8-85e4-e0d4200bc411.png)

solutions

![image](https://user-images.githubusercontent.com/22321977/48877326-bca2c600-ee3d-11e8-859e-b356d84d0b19.png)

![image](https://user-images.githubusercontent.com/22321977/48877337-c6c4c480-ee3d-11e8-919f-593ebfdadae4.png)

![image](https://user-images.githubusercontent.com/22321977/48877343-ce846900-ee3d-11e8-9987-7b583e2ef02f.png)

![image](https://user-images.githubusercontent.com/22321977/48877349-d47a4a00-ee3d-11e8-95a9-7137cb61ed10.png)
